components/giftbox.vue

```diff
@@ -0,0 +1,280 @@
+<template>
+  <div class="giftbox-wrapper">
+    <div class="giftbox-button" @click="showModal">
+      <fa icon="gift" class="giftbox-icon" />
+    </div>
+    <b-modal
+      ref="giftbox-modal"
+      class="giftbox-modal"
+      size="xl"
+      title="Nieuw in onze API en widgets"
+      :scrollable="false"
+      hide-footer
+    >
+      <div class="features-wrapper">
+        <div class="features-list">
+          <div
+            v-for="feature in features"
+            :key="feature.uid"
+            :class="[
+              'feature-title-block',
+              { selected: feature.uid === selectedFeature.uid },
+              { seen: seenFeatures.has(feature.uid) },
+            ]"
+            @click="switchFeature(feature.uid)"
+          >
+            <div class="notification-icon-wrapper">
+              <fa
+                v-if="!seenFeatures.has(feature.uid)"
+                icon="eye"
+                class="eye-icon"
+              />
+              <fa v-else icon="eye-slash" class="eye-icon" />
+            </div>
+            <p>{{ feature.title }}</p>
+          </div>
+        </div>
+        <div v-if="selectedFeature" class="features-detail">
+          <div class="title-calltoaction-container">
+            <h6>{{ selectedFeature.title }}</h6>
+            <a
+              :href="selectedFeature.callToAction"
+              :alt="selectedFeature.callToActionLabel"
+              target="_blank"
+              class="btn-call-to-action"
+            >
+              {{ selectedFeature.callToActionLabel }}
+            </a>
+          </div>
+          <a
+            v-if="selectedFeature.image"
+            :href="selectedFeature.image"
+            target="_blank"
+          >
+            <img
+              :src="selectedFeature.image"
+              :alt="selectedFeature.title"
+              class="feature-image"
+            />
+          </a>
+          <!-- eslint-disable-next-line vue/no-v-html -->
+          <div class="feature-content" v-html="selectedFeature.body" />
+        </div>
+        <div v-else class="features-detail">
+          <p>Geen nieuwe features te melden.</p>
```

Comment:
Use i18n 🌍  for translations. UDB is availabale in both french and dutch

Reply:
True, I mentioned the same in my call with @simonpubliq earlier today. We weren't sure if it would be possible already without the i18n PR merged, but in the end the function name to call in templates to do translations didn't change after all (at first it changed from `$t()` to `$i18n.t()`, but that was reverted by using `nuxt-i18n` instead of `i18next`) so we agreed he could already do it here already.